### ISSUE TITLE
evaljs is blocking the bot for long running tasks

### ISSUE THREAD

**Issue by pfilippi24** [AWulkan/Mr-Roboto#2]
Think about using something like https://www.npmjs.com/package/webworker-threads

**Comment by AWulkan** [comment]
Good idea. I am planning on making it handle these scenarios better later on, one way or another. I also might switch over to the official Discord.js framework offering async/await: 
https://discord.js.org/#/docs/commando/master/general/welcome
